plugins/profiles.koplugin/main.lua

```diff
@@ -337,10 +337,19 @@ function Profiles:getProfileFromCurrentDocument(new_name)
             "kopt_quality",
         }
     end
+    local setting_needs_arg = {
+        ["kopt_trim_page"]       = true,
+        ["kopt_zoom_mode_genus"] = true,
+        ["kopt_zoom_mode_type"]  = true,
+    }
 
     local profile = { settings = { name = new_name, order = document_settings } }
     for _, v in ipairs(document_settings) do
-        profile[v] = self.document.configurable[self.ui.rolling and v or v:sub(6)]
+        local value = self.document.configurable[self.ui.rolling and v or v:sub(6)]
```

Comment:
Looking at the code around, I couldn't find the answer: what are these 5 first chars removed by this `v:sub(6)` ?
I guess it's `kopt_` (looking at one of my metadata.pdf.lua) ? Might be worth a comment around here.
Btw, no issue in there with profiles with your cre/rolliing settings deduplication and the main and unique setting now being `copt_something`?